sentences/pt/light_HassTurnOff.yaml

```
@@ -3,14 +3,14 @@ intents:
   HassTurnOff:
     data:
       - sentences:
-          - "<desliga> [(todas | todos)] [(o | os | a | as)] <luz> <na_zona>"
+          - "<desliga> [tod[a|o]s] [(o[s]| a[s])] <luz> <na_zona>"
```

```suggestion
          - "<desliga> [tod(as|os)] [(o[s]|a[s])] <luz> <na_zona>"
```

`tod[a|o]s` would match tods`.
Also, lets use `(as|os)` in `todos/todas`, to make the rule a bit simpler to read.